app/Services/Category.php

```diff
@@ -10,4 +12,30 @@ abstract class Category
     const SEARCH = 'Search';
     const STORAGE = 'Storage';
     const TOOLS = 'Tools';
+
+    public static function all(): array
```

Comment:
I think this would more be a method on the Service class named `allByCategory` or something like that?

Reply:
`all()` now returns:

```php
$result = [
    'beanstalkd' => "Cache",
    'clickhouse' => "Database",
    'dynamodb' => "Cache",
    'elasticsearch' => "Search",
    'eventstoredb' => "Database",
    'expose' => "Tools",
    'influxdb' => "Database",
    'mailhog' => "Mail",
    'mariadb' => "Database",
    'meilisearch' => "Search",
    'memcached' => "Cache",
    'minio' => "Storage",
    'mongo' => "Database",
    'mssql' => "Database",
    'mysql' => "Database",
    'neo4j' => "Database",
    'postgis' => "Database",
    'postgresql' => "Database",
    'redis' => "Cache",
    'sftp' => "Storage",
    'sqs' => "Cache"
];
```

so the right name (for me) is `mapByService()`

Reply:
Sure, but I still think this is more of a list of services than a list of categories, right?

Reply:
whoops sorry 🙈 I misunderstood!
Apologies, you're right, it is a method for Service class.